package.json

```diff
@@ -59,8 +60,8 @@
     "nsp": "^2.6.3",
     "pre-commit": "^1.2.2",
     "standard-version": "^4.0.0",
-    "webpack": "^2.3.2",
-    "webpack-defaults": "^1.0.1"
+    "webpack": "^3.0.0",
```

Comment:
Remove this and add it as a `peerDependency`. The CI configs install the proper versions of webpack & developers can install if & when needed.